docs/docs-ref-autogen/excel/excel/excel.range.yml

```
@@ -1879,10 +1882,10 @@ methods:
     package: excel!
     fullName: getImage()
     summary: >-
-      Renders the range as a base64-encoded png image. **Important**: This API
-      is currently unsupported in Excel for Mac. Visit [OfficeDev/office-js
-      Issue \#235](https://github.com/OfficeDev/office-js/issues/235) for the
-      current status.
+      Renders the range as a base64-encoded png image. *Important**: This API is
+      currently unsupported in Excel for Mac. Visit \[OfficeDev/office-js Issue
```

This change seems to have added extra backslashes ( \ ) before the link and at the end of the link on the next line. 
```suggestion
      currently unsupported in Excel for Mac. Visit [OfficeDev/office-js Issue
```

---

@alison-mk Same here - you can verify the behavior by checking the review site for this branch